recipe/build.sh

```diff
@@ -29,6 +41,18 @@ for i in `ls`; do
         elif [[ $i == "include" ]]; then
             cp -rv $i ${PREFIX}/${targetsDir}
         elif [[ $i == "nvvm" ]]; then
+            for j in `find "${i}"`; do
+                if [[ "${j}" =~ /bin/.*$ ]]; then
+                    echo patchelf --force-rpath --set-rpath "\$ORIGIN/../lib64:\$ORIGIN/../../lib:\$ORIGIN/../../${targetsDir}/lib" "${j}" ...
```

Comment:
Suggesting a comment here to make it easier to understand what this is checking

```suggestion
                    # Adds the following paths relative to `$PREFIX` to the `RPATH`: `nvvm/lib64`, `lib`, `${targetsDir}/lib`
                    echo patchelf --force-rpath --set-rpath "\$ORIGIN/../lib64:\$ORIGIN/../../lib:\$ORIGIN/../../${targetsDir}/lib" "${j}" ...
```